### PR TITLE
add README.md,ui-ingress.yaml for charts

### PR DESCRIPTION
**Pull request by innerpeacez**

### CODE REVIEW

File: helm/charts/6/6.0.0-GA/skywalking/README.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+> **Tip**: You can use the default [values.yaml](values.yaml)
+
+### RBAC Configuration
+Roles and RoleBindings resources will be created automatically for `OAP` .
+
+> **Tip**: You can refer to the default `oap-role.yaml` file in [templates](templates/) to customize your own.
+
+### ConfigMap Files
+Skywalking is configured through [aplication.yaml](https://prometheus.io/docs/operating/configuration/). This file (and any others listed in `Files`) will be mounted into the `skywalking-oap` pod.
```
Comment (hanahmily) [inline helm/charts/6/6.0.0-GA/skywalking/README.md]: typo of application.yaml's url

**Comment by innerpeacez** [comment]
PTAL @hanahmily ,thanks

**Comment by innerpeacez** [comment]
This pr should not be merged, so I will add it directly to this changes requested, about the time zone configuration.

File: helm/charts/6/6.0.0-GA/skywalking/values.yaml [hunk 2]
```diff
@@ -105,3 +105,5 @@ ui:
     requests:
      # cpu: 100m
      memory: 1Gi
+
+TZ: Asia/Shanghai
```
Comment (wu-sheng) [inline helm/charts/6/6.0.0-GA/skywalking/values.yaml]: Please don't set TZ to China. We don't know the user timezone. UTC 0 as default is better.

Comment (innerpeacez) [reply helm/charts/6/6.0.0-GA/skywalking/values.yaml]: You have thought very carefully. @wu-sheng

**Pull request merged**